articles/sentinel/normalization.md

```diff
@@ -107,6 +107,43 @@ Each schema field has a type. Some have built-in, Azure Log Analytics types such
+| **EventEndTime**        | Mandatory   | Alias      |      Alias to the [TimeGenerated](#timegenerated) field.    |
+|  <a name=eventtyoe></a>**EventType**           | Mandatory   | Enumerated |    Describes the operation reported by the record. Each schema documents the list of values valid for this field. |
+| **EventSubType** | Optional | Enumerated | Describes a subdivision of the operation reproted in [EventType](#eventtype). Each schema documents the list of values valid for this field. |
+| **EventResult** | Mandatory | Enumerated | One of the following values: **Success**, **Partial**, **Failure**, **NA** (Not Applicable).<br> <br>The value may be provided in the source record using different terms, which should be normalized to these values. Alternatively, the source may provide only the [EventResultDetails](#eventresultdetails) field, which should be analyzed to derive the EventResult value.<br><br>Example: `Success`|
+| <a name=eventresultdetails></a>**EventResultDetails** | Mandatory | Alias | Reason or details for the result reported in the **_EventResult_** field. Each schema documents the list of values valid for this field.<br><br>Example: `NXDOMAIN`|
+| **EventOriginalUid**    | Optional    | String     |   A unique ID of the original record, if provided by the source.<br><br>Example: `69f37748-ddcd-4331-bf0f-b137f1ea83b`|
+| **EventOriginalType**   | Optional    | String     |   The original event type or ID, if provided by the source. For example, this field will be used to store the original Windows event ID.<br><br>Example: `4624`|
+| <a name ="eventproduct"></a>**EventProduct**        | Mandatory   | String     |             The product generating the event. <br><br>Example: `Sysmon`<br><br>**Note**: This field may not be available in the source record. In such cases, this field must be set by the parser.           |
+| **EventProductVersion** | Optional    | String     | The version of the product generating the event. <br><br>Example: `12.1`      |
+| **EventVendor**         | Mandatory   | String     |           The vendor of the product generating the event. <br><br>Example: `Microsoft`  <br><br>**Note**: This field may not be available in the source record. In such cases, this field must be set by the parser.  |
+| **EventSchemaVersion**  | Mandatory   | String     |    The version of the schema. Each schema documents its current version.         |
+| **EventReportUrl**      | Optional    | String     | A URL provided in the event for a resource that provides additional information about the event.|
+| **Dvc** | Mandatory       | String     |               A unique identifier of the device on which the event occurred. <br><br>Example: `ContosoDc.Contoso.Azure` <br><br>This field may alias any of [**DvcId**](#dvcid), **[DvcHostname](#dvchostname)** or **[DvcIpAddr](#dvcipaddr)** fields. For cloud sources, for which there is not apparent device, use the same value as [**Event Product**](#eventproduct)           |
```

Comment:
`This field may alias the [**DvcId**](#dvcid), **[DvcHostname](#dvchostname)**, or **[DvcIpAddr](#dvcipaddr)** fields. For cloud sources, for which there is no apparent device, use the same value as [**Event Product**](#eventproduct).`